spec/models/service/retirement_management_spec.rb

```diff
@@ -53,21 +68,20 @@
   it "#retire_now with userid" do
     expect(@service.retirement_state).to be_nil
     event_name = 'request_service_retire'
-    event_hash = {:service => @service, :type => "Service",
-                  :retirement_initiator => "user", :userid => "freddy"}
+    event_hash = {:userid => 'admin', :service => @service, :type => "Service"}
```

Comment:
```suggestion
    event_hash = {:userid => user.id, :service => @service, :type => "Service"}
```

not sure about this one